plugins/modules/launchd.py

```diff
@@ -30,6 +30,12 @@
       - Name of the service.
       type: str
       required: true
+    plist:
+      description:
+      - Name of the V(.plist) file for the service.
+      - Defaults to V({name}.plist).
+      type: str
+      version_added: 10.1.0
```

Comment:
Optimistic concurrency: that the PR will be merged within the right window.

Reply:
10.1.0 should be fine!